examples/client/getAccountInfo.ts

```diff
@@ -0,0 +1,67 @@
+import { AccountAddress, AccountInfo } from '@concordium/common-sdk';
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn example getAccountInfo
+
+  Options
+    --help,      -h  Displays this message
+    --endpoint,  -e  Specify endpoint of the form "address:port"
+    --account,   -a  An account address to get info from
+    --blockhash, -b  A blockhash to query the info from
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            endpoint: {
+                type: 'string',
+                shortFlag: 'e',
+                default: 'node.testnet.concordium.com:20000',
+            },
+            account: {
+                type: 'string',
+                short: 'a',
+                default: '3kBx2h5Y2veb4hZgAJWPrr8RyQESKm5TjzF3ti1QQ4VSYLwK1G',
+            },
+            blockhash: {
+                type: 'string',
+                short: 'b',
+                default:
+                    'fe88ff35454079c3df11d8ae13d5777babd61f28be58494efe51b6593e30716e',
```

Comment:
We should not have hardcoded blocks like that.

The behaviour that could make sense is to default to "last finalized" if the block is not given.